MapboxNavigation/RouteMapViewController.swift

```diff
@@ -687,6 +687,17 @@ extension RouteMapViewController: NavigationViewDelegate {
                 return
         }
         
+        let roadName = delegate?.mapViewController(self, roadNameAt: location)
+        guard roadName == nil else {
+            if let roadName = roadName {
+                navigationView.wayNameView.text = roadName
+                navigationView.wayNameView.isHidden = roadName.isEmpty
+            }
+            return
+        }
+        
+        navigationView.wayNameView.isHidden = true
```

Comment:
Is it still necessary to hide the label at this point? This is the preexisting code branch, which should already be working correctly.